ash/src/entry.rs

```diff
@@ -238,11 +238,18 @@ impl Entry {
     /// <https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/vkEnumerateInstanceExtensionProperties.html>
     pub fn enumerate_instance_extension_properties(
         &self,
+        p_layer_name: Option<&CStr>,
```

Comment:
```suggestion
        layer_name: Option<&CStr>,
```

We don't commonly use the `p_` prefix for "safe" rusty items.